web/admin/src/pages/orders/index.vue

```diff
@@ -73,21 +157,129 @@ export default defineComponent({
         text: '伝票番号',
         value: 'payment.paymentId',
       },
+      {
+        text: 'Actions',
+        value: 'actions',
+        sortable: false,
+      },
     ]
 
     const handleUpdatePage = async (page: number) => {
       updateCurrentPage(page)
       await orderStore.fetchOrders(itemsPerPage.value, offset.value)
     }
 
+    const getStatusColor = (status: PaymentStatus): string => {
+      switch (status) {
+        case PaymentStatus.UNKNOWN:
+          return 'unkown'
+        case PaymentStatus.UNPAID:
+          return 'secondary'
+        case PaymentStatus.PENDING:
+          return 'secondary'
+        case PaymentStatus.AUTHORIZED:
+          return 'info'
+        case PaymentStatus.PAID:
+          return 'primary'
+        case PaymentStatus.REFUNDED:
+          return 'primary'
+        case PaymentStatus.EXPIRED:
+          return 'error'
+        default:
+          return 'unkown'
+      }
+    }
+
+    const getStatus = (status: PaymentStatus): string => {
+      switch (status) {
+        case PaymentStatus.UNKNOWN:
+          return '不明'
+        case PaymentStatus.UNPAID:
+          return '未払い'
+        case PaymentStatus.PENDING:
+          return '保留中'
+        case PaymentStatus.AUTHORIZED:
+          return 'オーソリ済み'
+        case PaymentStatus.PAID:
+          return '支払い済み'
+        case PaymentStatus.REFUNDED:
+          return '返金済み'
+        case PaymentStatus.EXPIRED:
+          return '期限切れ'
+        default:
+          return '不明'
+      }
+    }
+
+    const getShippingMethod = (shippingMethod: DeliveryType): string => {
+      switch (shippingMethod) {
+        case DeliveryType.UNKNOWN:
+          return '不明'
+        case DeliveryType.NORMAL:
+          return '通常便'
+        case DeliveryType.REFRIGERATED:
+          return '冷蔵便'
+        case DeliveryType.FROZEN:
+          return '冷凍便'
+      }
+    }
+
+    const openImportDialog = (): void => {
+      importDialog.value = true
+    }
```

Comment:
こんな感じのメソッド作れば共通化できそう

```suggestion
    const toggleImportDialog = (): void => {
      importDialog.value = !importDialog.value
    }
```